### ISSUE TITLE
Display video category contents sorted by series

### ISSUE THREAD

**Issue by Huxely** [hirsivaja/plugin.video.areena#7]
It would clean up the UI very much If when browsing through different categories like Shows - Foreign shows or Comedy, that only "non series"-videos would be on the root directory of the category, and all "series"-videos would be placed inside folders that are named by their corresponding series title (and of course shown before the "non series" videos). Could this be achieved by using the part_of_series tag and remove these from the category root, and create a folder based on the tag, and locate the videos there?

So basically, when retrieving category videos, first list all the part_of_series tags, create folders for these, then hide videos based on these folders and display the rest. Then when user selects the folder, list videos based on the part_of_series tag.

**Comment by hirsivaja** [comment]
I tried this and the problem is that to be able to list all the series for the category we need to retrieve all the items for that category from the Yle. And to make it even slower Yle limits results per one query to 100. So for some categories that contain thousands of programs it takes tens of seconds to query the needed information.

Based on my efforts on this it is not practical to implement until Yle adds support to the API to list series by category. If you can think of a better way to implement this I can give another go or better yet if you can implement this yourself I am happy to test your pull request!
